trade-finance-manager-ui/server/nunjucks-configuration/filter-formatUnixTimestamp.ts

```diff
@@ -0,0 +1,15 @@
+import { format, isValid, toDate } from 'date-fns';
+
+/**
+ * @param isoDateStr date string formatted as fromFormat
+ * @param toFormat date format using {@link https://date-fns.org/v3.3.1/docs/format | unicode tokens}
+ * @returns date formatted as toFormat or 'Invalid date' if can't parse isoDateStr
+ */
+export const formatUnixTimestamp = (timestamp: number, toFormat: string) => {
+  if (!timestamp) {
+    return 'Invalid date';
+  }
+
+  const date = toDate(timestamp);
+  return isValid(date) ? format(date, toFormat) : 'Invalid date';
```

Comment:
Aware it's the existing pattern, is there a better way to handle invalid data in nunjucks files as opposed to returning an arbitrary 'invalid date' string?

Reply:
Hmm yes, I think it would be a case of either making both: 
- This return undefined
- And `dashIfEmpty` to stop handling the strange arbitrary string cases.
at the same time. 
But I don't think that's feasible in this PR / would be inconsistent across other services. 

So maybe one for another time and leave as is👍